Dockerfile

```diff
@@ -9,10 +11,14 @@ ADD demo/inventory /runner/inventory
 # Install Ansible Runner
 RUN yum-config-manager --add-repo https://releases.ansible.com/ansible-runner/ansible-runner.el7.repo && \
 	yum install -y epel-release && \
-	yum install -y python-pip ansible-runner bubblewrap sudo rsync openssh-clients sshpass && \
+	yum install -y python-pip ansible-runner bubblewrap sudo rsync openssh-clients sshpass \
+			gcc gcc-c++ make python-devel libffi-devel openssl-devel && \
```

Comment:
What do we need a full toolchain for here?

Reply:
that smells like the python-cryptography dependency of ansible attempting to be built natively rather than using the system package. This should be fixed to ensure that you're pulling the system python-cryptography build

Reply:
@matburt @wenottingham thanks for reviewing and providing your feedback, yes the issue is because of the cryptography module. the "pip" is not yet supported on ppc64le, as also, confirmed from https://cryptography.io/en/latest/installation/. 

So, it can either be built from source, which happens by default if the necessary *-devel packages are present in the container, hence I installed and later removed them.

The other option, I think is to try and go via the "conda" route as the requisite packages are present there https://anaconda.org/conda-forge/cryptography/files?channel=main , any thoughts here?

Reply:
That doesn't make sense. If you're using the same CentOS base, a python-cryptography build already exists at the system RPM level. Use that.

Reply:
yes, missed that, will make the change, thank you

Reply:
@wenottingham @matburt have updated, please review, thank you!

Reply:
any thoughts? thank you!!

Reply:
On the surface this is fine I reckcon, we just don't have any need or tooling for this so it's not possible for us to generate or test official images based on this.

Reply:
Thank you @matburt do you think it would make sense to merge the changes so that folks who need Power images can still build it off the official source for now? For generation of official images, if I may ask, is there an internal CI that you are using? We have support for Power on Travis today, so I can help setup things there or we can help provide Power nodes and help add them to your setup? We can work on enhancing the tooling, look at any other alternatives you suggest as well.